src/SelectMenu/SelectMenuStyles.js

```diff
@@ -0,0 +1,287 @@
+import {keyframes, css} from 'styled-components'
+import {get} from '../constants'
+
```

Comment:
[Minor] Naming this file `SelectMenuStyles.js` makes me think that is exports a React component called `SelectMenuStyles`. IMO `selectMenuStyles.js` would be a little more clear. 

Also, I wouldn't be opposed to colocating these styles with the components that use them. Is there a reason you chose to pull them out into this file?

Reply:
Yeah, I originally did that, but some of the components had a ton of styles and I felt like it was making the component file a little messy... but I might move them back because I've noticed it is making it harder to remember to update the styles if we change a prop name that's used in the CSS or something like that

Reply:
moved them out, it felt good 😂 👍